packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -92,11 +92,28 @@ const formattedPlanTypes: Record<PlanType, string> = {
   [PlanType.OnPrem]: 'Ops Manager',
 } as const;
 
+const onKeyDown = (e: React.KeyboardEvent, callback: Function) => {
```

Comment:
The callback will always be the useState setter function, so we can just pass that in directly.

Reply:
What do you mean?